src/Avatar/Avatar.tsx

```diff
@@ -1,8 +1,18 @@
-import React from 'react';
-import propTypes from 'prop-types';
+import React, { forwardRef } from 'react';
 import styled from 'styled-components';
+import { CommonStyledProps } from '../types';
 
-const StyledAvatar = styled.div`
+type AvatarProps = {
+  alt?: string;
+  children?: React.ReactNode;
+  noBorder?: boolean;
+  size?: string | number;
+  square?: boolean;
+  src?: string;
+} & React.HTMLAttributes<HTMLDivElement> &
+  CommonStyledProps;
+
+const StyledAvatar = styled.div<AvatarProps & { size?: string }>`
```

Comment:
I think we shouldn't put AvatarProps to StyledAvatar, as most properties on that type aren't actually passed to StyledAvatar. StyledAvatar only needs square, size, noBorder

Reply:
Done. It also needed `src` which wasn't being passed so I included it as well.